installer/install-image

```diff
@@ -26,6 +26,8 @@
 #
 
 . /kayak/lib/install_help.sh
+
+FailIfRpoolExists
```

Comment:
Unfortunately, this would break the way that a lot of people use Kayak today (including me) although I understand the risk that you are mitigating here.

If you want to do this, you can put it at the top of your kayak configuration file, before the usual `BuildRpool` line. The configuration files are just bash.

Reply:
I'm sorry I did not mean to include this second commit in the pull request but just the first commit before it. I thought that the pull request (which I did a day before this second commit) would only include the first. I did not expect consideration of this second commit as it would clearly break established behavior. I'm sorry the second commit got included in the request.